src/graph/backend/dnnl/patterns/sdp.cpp

```diff
@@ -436,17 +436,21 @@ DNNL_BACKEND_REGISTER_PATTERN_MATCHER_PASS(dnnl, sdp_with_compressed_v_fusion)
                 [](const std::shared_ptr<pb_graph_t> &pgraph) -> void {
                     auto matmul_qk = pgraph->append_op(graph::op_kind::MatMul);
 
-                    auto fscore_scale = pgraph->append_alternation(
-                            {graph::op_kind::Divide, graph::op_kind::Multiply},
-                            {in_edge(0, matmul_qk, 0)});
```

Comment:
Thanks for the fix. May need to support optional scale in the pattern `sdp_with_compressed_kv_fusion` as well.